creating_diverse_inclusive_communities.adoc

```diff
@@ -1,6 +1,6 @@
 = Creating diverse and inclusive communities
 Authors: Lauren Maffeo <lauren@address.tld>
```

Comment:
Be sure to add your preferred address here if you'd like!

Reply:
Is this email address going into the published book? If so, I'd rather leave it out.

Reply:
As far as I know, it will be included in file metadata but not more widely available. Nevertheless, it's not required and I'm certainly fine with omitting it at your request.